Service/InstallationService.php

```diff
@@ -28,10 +24,14 @@ class InstallationService implements InstallerInterface
     ];
 
     public const ENDPOINTS = [
-        ['path' => 'stuf/zds', 'throws' => ['vrijbrp.zds.inbound'], 'name' => 'zds-endpoint']
+        ['path' => 'stuf/zds', 'throws' => ['vrijbrp.zds.inbound'], 'name' => 'zds-endpoint', 'methods' => []],
     ];
 
     public const ACTION_HANDLERS = [
```

Comment:
Still not a fan of this way of creating Actions...
We can actually use action.json files now

Reply:
https://github.com/CommonGateway/GeboorteVrijBRPBundle/blob/a36ee0e43e3dc9842feac28247913733714a1a38/Action/vrijbrp.ZgwToVrijbrpAction.action.json

Reply:
This is an endpoint ;), agree on actions, but speed etc.

Reply:
My comment is on line 34 | 30
public const ACTION_HANDLERS = [

And i get that, this is just a comment not a request to change it right now, approved!